<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>caDNAno</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeview">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="CustomQGraphicsView" name="sliceGraphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::NoDrag</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::NoAnchor</enum>
       </property>
       <property name="resizeAnchor">
        <enum>QGraphicsView::AnchorViewCenter</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::MinimalViewportUpdate</enum>
       </property>
      </widget>
      <widget class="CustomQGraphicsView" name="pathGraphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>4</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::WheelFocus</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::MinimalViewportUpdate</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent_files">
     <property name="title">
      <string>Open recent files</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export...</string>
     </property>
     <addaction name="actionSVG"/>
     <addaction name="actionX3D"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent_files"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_a_Copy"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="leftToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Slice Tools</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea|Qt::RightToolBarArea</set>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSliceSelect"/>
   <addaction name="actionSliceMove"/>
   <addaction name="separator"/>
   <addaction name="actionSliceFirst"/>
   <addaction name="actionSliceLast"/>
   <addaction name="actionRenumber"/>
   <addaction name="actionDeleteLast"/>
  </widget>
  <widget class="QToolBar" name="topToolBar">
   <property name="windowTitle">
    <string>topToolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSVG"/>
   <addaction name="separator"/>
   <addaction name="actionNewHoneycombPart"/>
   <addaction name="actionNewSquarePart"/>
  </widget>
  <widget class="QToolBar" name="rightToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Path Tools</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPathSelect"/>
   <addaction name="actionPathForce"/>
   <addaction name="actionPathMove"/>
   <addaction name="actionPathBreak"/>
   <addaction name="actionPathErase"/>
   <addaction name="actionPathPencil"/>
   <addaction name="actionPathInsert"/>
   <addaction name="actionPathSkip"/>
   <addaction name="actionPathPaint"/>
   <addaction name="separator"/>
   <addaction name="actionAutoStaple"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/filetools/new</normaloff>:/filetools/new</iconset>
   </property>
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/filetools/open</normaloff>:/filetools/open</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/filetools/save</normaloff>:/filetools/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_a_Copy">
   <property name="text">
    <string>Save a Copy...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionSVG">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/filetools/svg</normaloff>:/filetools/svg</iconset>
   </property>
   <property name="text">
    <string>SVG</string>
   </property>
  </action>
  <action name="actionX3D">
   <property name="text">
    <string>X3D</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionNewHoneycombPart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/parttools/new-honeycomb</normaloff>:/parttools/new-honeycomb</iconset>
   </property>
   <property name="text">
    <string>Honeycomb</string>
   </property>
   <property name="toolTip">
    <string>Click to add new part with honeycomb lattice</string>
   </property>
  </action>
  <action name="actionPathBreak">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/break</normaloff>:/pathtools/break</iconset>
   </property>
   <property name="text">
    <string>Break Tool</string>
   </property>
   <property name="iconText">
    <string>Break</string>
   </property>
   <property name="toolTip">
    <string>Click to add part with honeycomb lattice</string>
   </property>
  </action>
  <action name="actionPathSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/select</normaloff>:/pathtools/select</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Selection Tool</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="actionPathForce">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/force</normaloff>:/pathtools/force</iconset>
   </property>
   <property name="text">
    <string>Force</string>
   </property>
   <property name="toolTip">
    <string>Forcibly add a crossover between unusual helices/bases. The first click defines the 3' side and the second click defines the 5' side.</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionSliceSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/select</normaloff>
     <selectedoff>:/slicetools/images/slice-edit.png</selectedoff>:/slicetools/select</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="shortcut">
    <string>Shift+V</string>
   </property>
  </action>
  <action name="actionSliceFirst">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/go first</normaloff>:/slicetools/go first</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
   <property name="toolTip">
    <string>Move the slice bar to the first position.</string>
   </property>
  </action>
  <action name="actionSliceLast">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/go last</normaloff>:/slicetools/go last</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
   <property name="toolTip">
    <string>Move the slice bar to the last position.</string>
   </property>
  </action>
  <action name="actionPathErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/erase</normaloff>:/pathtools/erase</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
   <property name="toolTip">
    <string>Select this tool and then click on a path to erase it from the design.</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionAutoStaple">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/autostaple</normaloff>:/pathtools/autostaple</iconset>
   </property>
   <property name="text">
    <string>Staple</string>
   </property>
   <property name="toolTip">
    <string>Click this button to generate a default set of staples.</string>
   </property>
  </action>
  <action name="actionPathPencil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/force</normaloff>:/pathtools/force</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
   <property name="toolTip">
    <string>Draw strands. Right-click to force non-standard crossover</string>
   </property>
   <property name="shortcut">
    <string>F, N</string>
   </property>
  </action>
  <action name="actionPathInsert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/insert</normaloff>:/pathtools/insert</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
   <property name="toolTip">
    <string>Insert extra bases at a specific position. Useful for creating curved and twisted designs.</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionNewSquarePart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/parttools/new-square</normaloff>:/parttools/new-square</iconset>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
   <property name="toolTip">
    <string>Click to add new part with square lattice</string>
   </property>
  </action>
  <action name="actionPathSkip">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/skip</normaloff>:/pathtools/skip</iconset>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
   <property name="toolTip">
    <string>Leave out specified bases during assigning scaffold and staple bases. Useful for making curved and twisted designs.</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionSliceMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/move</normaloff>:/slicetools/move</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move (M)</string>
   </property>
  </action>
  <action name="actionPathMove">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/pathtools/move</normaloff>:/pathtools/move</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move (M)</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionRenumber">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/renumber</normaloff>:/slicetools/renumber</iconset>
   </property>
   <property name="text">
    <string>Renum</string>
   </property>
   <property name="toolTip">
    <string>Renumber Slice helices according to helix ordering in Path panel.</string>
   </property>
  </action>
  <action name="actionDeleteLast">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/slicetools/del-last</normaloff>:/slicetools/del-last</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Remove the highest-numbered helix if it does not connect to any other helices.</string>
   </property>
  </action>
  <action name="actionPathPaint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/pathtools/paint</normaloff>:/pathtools/paint</iconset>
   </property>
   <property name="text">
    <string>Paint</string>
   </property>
   <property name="toolTip">
    <string>Use to color staples.</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>customqgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
